modules/emulator/R/GaussProcess.R

```diff
@@ -256,17 +256,17 @@ GaussProcess <- function(x, y, isotropic = TRUE, nugget = TRUE, method = "bayes"
     aden <- aden + mvtnorm::dmvnorm(as.vector(W), rep(0, n.unique), S, log = TRUE)
     if (is.numeric(anum) && 
         is.finite(anum) && 
-        exp(anum - aden) > runif(1)) {
+        exp(anum - aden) > stats::runif(1)) {
       tauw <- taustar
       S <- Sstar
     }
     tauwgibbs[g, ] <- tauw
-    tauwjump <- update(tauwjump, tauwgibbs)
+    tauwjump <- stats::update(tauwjump, tauwgibbs)
```

Comment:
This is actually a call to the `update` function thats within this package, not the one in stats